build/device.mk

```diff
@@ -278,9 +278,6 @@ ENABLE_CAMERA_SERVICE=true
 ENABLE_EVS_SAMPLE=true
 $(call inherit-product, vendor/xen/evs/evs.mk)
 
-PRODUCT_PRODUCT_PROPERTIES += \
```

Comment:
Hi @DKlimenko, you've added this directive here in the same PR - https://github.com/xen-troops/android_device_xenvm/pull/92/commits/ec9935a0be8106cf948fb88f1c7ba01810167a03#diff-a77da357f7efbbcf77d2baa145a6bb6da7bd54e2a956b94727149dce6849d107R281

But now you remove it within the same PR. It is not valid. Please, squash not-needed lines to not pollute the diff with useless changes.

Reply:
Hi. Squashed commits. Not actual any more